CHANGELOG.md

```diff
@@ -1,7 +1,10 @@
 ## UNRELEASED
 
 FEATURES:
-* Peering: Add support for `PeerThroughMeshGateways` in Mesh CRD. [[GH-1478](https://github.com/hashicorp/consul-k8s/pull/1478)]
+* Peering:
+  * Add support for `PeerThroughMeshGateways` in Mesh CRD. [[GH-1478](https://github.com/hashicorp/consul-k8s/pull/1478)]
+  * move support for customizing the server addresses in peering token generation. Instead, mesh gateways should be used
```

Comment:
I am not sure but should the token changes from the helm chart be listed as a breaking change?

Reply:
Yup added it to breaking instead!